### PR TITLE
added sql script

### PR DESCRIPTION
**Pull request by azizbekxm**
### College sharing instance table
telegram-cloud-photo-size-2-5418291979562900756-y

### Consortium inventory instance table
telegram-cloud-photo-size-2-5418291979562900764-y

### SQL Script run
telegram-cloud-photo-size-2-5418291979562900784-y

### Result in consortia sharing instance table
telegram-cloud-photo-size-2-5418291979562900785-y

### CODE REVIEW

File: src/main/resources/fix_mod_consortia_table.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    DELETE FROM college_mod_consortia.sharing_instance
+    WHERE target_tenant_id = target_tenant;
+
+    -- Select records from the  <target_tenant>_mod_inventory_storage.instance table where the "source" is 'CONSORTIUM-MARC'
+    WITH record_from_target_tenant as (
+        SELECT id FROM consortium_mod_inventory_storage.instance
+        WHERE jsonb->>'source' = 'CONSORTIUM-MARC'
+    )
+    -- Insert each record into <source_tenant>_mod_consortia
+    INSERT INTO college_mod_consortia.sharing_instance (id, instance_id, source_tenant_id, target_tenant_id, status)
```
Comment (SerhiiNosko) [inline src/main/resources/fix_mod_consortia_table.sql]: we should insert records about sharing instances only in central tenant sharing_instance table

File: src/main/resources/fix_mod_consortia_table.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
+DO $$
+DECLARE
+    target_tenant TEXT:= 'consortium';
+    source_tenant TEXT:= 'college';
+    record RECORD;
```
Comment (SerhiiNosko) [inline src/main/resources/fix_mod_consortia_table.sql]: please add readme how to use this fuction
ANd I will provide also example how to use API part

File: src/main/resources/fix_mod_consortia_table.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
+DO $$
+DECLARE
+    target_tenant TEXT:= 'consortium';
```
Comment (SerhiiNosko) [inline src/main/resources/fix_mod_consortia_table.sql]: need to swap, in their example targetTenant it is member tenant

Comment (SerhiiNosko) [reply src/main/resources/fix_mod_consortia_table.sql]: better to rename param in our script as well to not have confusion to 'memberTenant'

File: src/main/resources/fix_mod_consortia_table.sql (hunk 1 above)
Comment (azizbekxm) [reply src/main/resources/fix_mod_consortia_table.sql]: thanks, fixed

File: src/main/resources/fix_mod_consortia_table.sql (hunk 2 above)
Comment (azizbekxm) [reply src/main/resources/fix_mod_consortia_table.sql]: added explanation

File: src/main/resources/fix_mod_consortia_table.sql (hunk 3 above)
Comment (azizbekxm) [reply src/main/resources/fix_mod_consortia_table.sql]: changed and used your suggestion

**Pull request merged**